### PR TITLE
[TW-714] Doc user shared auth, OAuth for on-prem, and multiple APIs to one repo

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- "Using a remote Git repository" page - Updated connecting to cloud and on-prem repos, added section about connecting multiple APIs to a repo
- "Setting up Postman" page - Updated "Connected accounts" section
- "Using installed apps in integrations" page - Added GitHub Enterprise and GitLab Self-Managed

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 1]
```diff
@@ -82,49 +84,134 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
...
 
-1. Select **APIs** in the sidebar and select an API.
-1. Under **Connect repository**, select **Connect** and select the type of repository you want to connect to.
-1. Enter the **Domain URL** of your repository. You can enter a new domain or select a domain you've used before from the list. Format the domain URL as follows:
+To connect to a repository hosted in GitHub Enterprise Server or GitLab Self-Managed, [create an installed app](#creating-an-installed-app) in Postman. To complete this process, you'll also need to create an OAuth app in GitHub Enterprise Server or GitLab Self-Managed. You need to complete this process once for a domain URL. After you create the installed app, anyone on the team can use it to [connect to repositories using the same domain URL](#connecting-an-api-using-an-installed-app).
+
+#### Creating an installed app
+
+[Installed apps](/docs/integrations/installed-apps/) streamline the process of adding integrations by enabling all members of a team to use the same stored authorization details. To create an installed app for GitHub Enterprise Server or GitLab Self-Managed, do the following:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:108]: According to the [style guide](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3978756522/Procedures#Lead-in-sentences), we should put lead-in sentences on their own line.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 2]
```diff
@@ -82,49 +84,134 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
...
+    * For **Authorization callback URL** or **Redirect URI**, enter the callback URL provided in Postman.
+
+1. After registering the OAuth app, copy the app's **Client ID** and **Client secret** and enter them in Postman.
+1. Select **Save** to finish creating the installed app.
+
+<img alt="Creating an installed app" src="https://assets.postman.com/postman-docs/v10/api-builder-add-installed-app-v10-12.jpg" width ="572px"/>
+
+#### Connecting an API using an installed app
+
+After you [create an installed app](#creating-an-installed-app), any API editor on the team can use it to connect an API to GitHub Enterprise Server or GitLab Self-Managed. To connect an API using an installed app, do the following:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:131]: According to the [style guide](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3978756522/Procedures#Lead-in-sentences), we should put lead-in sentences on their own line.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:108]: Great, thanks for spotting this!

**Comment by jatin3893** [comment]
Screenshot 2023-03-08 at 10 25 48 PM

Sharing feedback as shared on Slack^

**Comment by jatin3893** [comment]
Do we need to specify any details like "This is supported from Postman v10.12 onwards" or something like that?

**Comment by mark-anthony-postman** [comment]
> Do we need to specify any details like "This is supported from Postman v10.12 onwards" or something like that?

@jatin3893 We generally don't do this, as the Learning Center docs are always for the latest version of Postman

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 3]
```diff
@@ -82,49 +84,138 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
...
+* `notifications-v1` branch
+    * `/postman/schemas` - Schemas for the `notifications-v1` API
+    * `/postman/collections` - Collections for the `notifications-v1` API
+* `notifications-v2` branch
+    * `/postman/schemas` - Schemas for the `notifications-v2` API
+    * `/postman/collections` - Collections for the `notifications-v2` API
 
-Postman securely stores your personal access token. After connecting to an on-premises repository, you can connect to other repositories using the same domain URL without having to re-enter your token. When connecting to a repository, select **Change** under **Authenticated Domain** if you want to use a different domain URL.
+You must use a unique branch + folder combination for each API you [connect to the repository](#connecting-to-a-repository). For example, if you connect one API to the `/postman/schemas` folder on the `notifications-v1` branch, you can't connect another API to the same folder on the same branch. You must use a different branch or a different schema folder.
```
Comment (jkonrath-postman) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:216]:
```suggestion
You must use a unique branch and folder combination for each API you [connect to the repository](#connecting-to-a-repository). For example, if you connect one API to the `/postman/schemas` folder on the `notifications-v1` branch, you can't connect another API to the same folder on the same branch. You must use a different branch or a different schema folder.
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Comment by mark-anthony-postman** [comment]
> Sharing feedback (same as the one on slack)

@jatin3893 Thanks for the great feedback! I've added your suggestions to the documentation and updated the PR.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 4]
```diff
@@ -79,52 +81,141 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
...
 
     * **GitHub Enterprise Server** - Use the base URL of your Git server, for example: `https://my-github-server.example.com`
     * **GitLab Self-Managed** - Use the base URL of your Git server, for example: `https://gitlab.example.com`
-    * **Azure DevOps Server** - Use the name and port of your Git server along with the collection, for example: `https://my-azure-server:8080/my-collection`
 
-1. Enter a **Personal access token** to access the repository. Ensure that the token has the required permissions as specified on the **Connect your repository** page.
+1. Create an OAuth app by following the instructions for your Git provider ([GitHub Enterprise Server](https://go.pstmn.io/github-oauth-app-setup) or [GitLab Self-Managed](https://go.pstmn.io/GitLab-OAuth-app-setup)).
+
+    * For **Homepage URL** enter: `https://www.postman.com`
+    * For **Authorization callback URL** or **Redirect URI**, enter the authorization callback URL provided in Postman.
```
Comment (shashankawasthi88) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:124]: Should we provide the URL here as well?

**Review by shashankawasthi88 — APPROVED** [review]
@mark-anthony-postman this looks good, but we should add the call back URL in docs as well/

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md (hunk 4 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:124]: @shashankawasthi88 Sure, that makes sense as long as the callback URL is always the same.

**Pull request merged**